src/lib/components/Footer/Footer.tsx

```diff
@@ -1,150 +1,263 @@
-import { Box, Grid, Link, Typography } from '@mui/material';
-import { Trans, useTranslation } from 'react-i18next';
-import { buildAssistanceURI } from '../../services/assistanceService';
-import PagoPaIconColored from '../icons/PagoPaIconColored';
+import { Footer as MuiItaliaFooter } from '@pagopa/mui-italia/dist/components/Footer/Footer';
+import i18n from '../../locale/locale-utils';
+import {
+  CompanyLinkType,
+  FooterLinksType,
+  LangCode,
+  LinkType,
+  PreLoginFooterLinksType,
+} from '../../model/Mui-italia-model';
 
-type Props = {
-  /** The email to which the assistance button will ask to send an email */
-  assistanceEmail?: string;
-  onExit?: (exitAction: () => void) => void;
+type FooterProps = {
+  loggedUser: boolean;
+  onExit?: (href: string, linkType: LinkType) => void;
+  onLanguageChanged?: (newLang: LangCode) => void;
 };
 
-/** SelfCare Footer component */
-const Footer = ({ assistanceEmail, onExit = (exitAction) => exitAction() }: Props) => {
-  const { t } = useTranslation();
+const companyLegalInfo = (
```

Comment:
create a new external file, FooterConfig or something like this, where to put all this config in order to have just the component here and in a separate file all the configuration
(This because there is a lot of configuration)